xbmc/windowing/win10/WinSystemWin10.h

```diff
@@ -101,6 +101,7 @@ class CWinSystemWin10 : public CWinSystemBase
 
 protected:
   bool CreateNewWindow(const std::string& name, bool fullScreen, RESOLUTION_INFO& res) override = 0;
+  void* GetHWContext() override = 0;
```

Comment:
I see no reason to add this.

Reply:
Seen it on the other and thought to do the same but you are right, by `CreateNewWindow` the CWinSystemBase is also defined with `=0` and then makes it sense.